.ai/auto-review/env_vars.py

```diff
@@ -30,8 +30,7 @@ def __init__(self):
             "token" : self.token,
             "base_ref" : self.base_ref,
             "pull_number" : self.pull_number,
-            "chat_gpt_token" : self.chat_gpt_token,
-            "chat_gpt_model" : self.chat_gpt_model,
+            "gemini_ai_token" : self.gemini_ai_token
         }
```

Comment:
34 : Incorrect splitting of `TARGET_EXTENSIONS` if it contains spaces : Use `self.target_extensions = [lang.strip() for lang in self.target_extensions.split(",")]` instead of `self.target_extensions.split(",")`